lib/gui/models/flash-state.js

```diff
@@ -220,3 +220,20 @@ exports.getLastFlashSourceChecksum = () => {
 exports.getLastFlashErrorCode = () => {
   return exports.getFlashResults().errorCode;
 };
+
+/**
+ * @summary Get last flash uuid
```

Comment:
IMHO this description might be a bit misleading - perhaps "Get current (or last) flash uuid" instead?